### PR TITLE
Add `sched_priority` and `logbanner` options.

### PR DESCRIPTION
**Pull request by jcpunk**
#### Pull Request (PR) description
This patch adds a way to set the chrony options `sched_priority` and `logbanner`.

#### This Pull Request (PR) fixes the following issues
None

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -251,6 +255,7 @@
   Stdlib::Unixpath $driftfile                                      = '/var/lib/chrony/drift',
   Variant[Boolean[false],Integer[1,15]] $local_stratum             = 10,
   Optional[String[1]] $log_options                                 = undef,
+  Optional[Integer] $logbanner                                     = undef,
```
Comment (root-expert) [inline manifests/init.pp:258]: `logbanner` can't have negative values.
```suggestion
  Optional[Integer[0]] $logbanner                                  = undef,
```

File: manifests/init.pp [hunk 2]
```diff
@@ -270,6 +275,7 @@
   Optional[String[1]] $mailonchange                                = undef,
   Float $threshold                                                 = 0.5,
   Boolean $lock_all                                                = false,
+  Optional[Integer] $sched_priority                                = undef,
```
Comment (root-expert) [inline manifests/init.pp:278]: Same here. Probably there's an upper bound as well? 100 maybe?
```suggestion
  Optional[Integer[0]] $sched_priority                             = undef,
```

**Review by root-expert — CHANGES_REQUESTED** [review]
Thanks for the PR!

Please add basic unit tests for the new parameters.

**Comment by jcpunk** [comment]
I've taken a stab at fixing the things you noted.  This look better?

**Review by root-expert — APPROVED** [review]

**Review by smortex — APPROVED** [review]

**Pull request merged**